.travis/install.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e
+set -x
+
+if [[ "$(uname -s)" == 'Darwin' ]]; then
+    sw_vers
+
+    git clone --depth 1 https://github.com/pyenv/pyenv ~/.pyenv
```

Comment:
You might want to cache this directory in travis and not retrieve it every time.